Stresser/EndpointController.h

```diff
@@ -4,6 +4,7 @@
 #include "EndpointEntity.h"
 
 #include "../Utils/AutoHandle.h"
+#include "../Utils/StringUtils.h"
```

Comment:
Move this to the source file.
This is part of the "include order" best practices.
Always put your include closer as you can to the place you are using them.
if you don't use: `... = StringUtils::wstringToString(....)` in the header file (which you don't) you should move this include to the source file.
This prevents potential bugs in compile time(corresponding to include, linking, etc)